### DISCUSSION TITLE
[Code view beta] Symbols' language support

### DISCUSSION THREAD

**Discussion by guidezpl** [community/community#38751] [Repositories]
### Select Topic Area

Product Feedback

### Body

What is the plan for language support for symbols? I just tried Dart and it seems to only do search-based symbol finding.

**Comment by ossanna16** [comment]
Hi there @guidezpl and welcome to our community. Thank you for the feedback!

**Comment by jfullstackdev** [comment]
what do you mean by Symbol language ? 

if that is a programming language, there is the dedicated group to work it out

https://github.com/github/linguist
